exonum-java-binding-core/rust/ejb-app/TUTORIAL.md

```diff
@@ -94,5 +94,24 @@ $ cargo run -- finalize testnet/sec.toml testnet/node.toml \
 ### Step 3. Run Configured Node
 
 ```$sh
-$ cargo run -- run -d testnet/db -c testnet/node.toml --public-api-address 127.0.0.1:3000
+$ cargo run -- run -d testnet/db -c testnet/node.toml --public-api-address 127.0.0.1:3000 
 ```
+
+There are few specific optional parameters here:
+- `--jvm-args-prepend` and `--jvm-args-append`: Additional parameters for JVM that prepend and
+ append the rest of arguments. Must not have a leading dash. For example, `Xmx2G` or `Xdebug`.
```

Comment:
Would you please remove `Xdebug` from all examples? This is a legacy JVM option that does nothing.

Reply:
Ok, removing them.